src/main/g8/build.sbt

```diff
@@ -12,7 +11,6 @@ lazy val root = (project in file(".")).
       "com.typesafe.akka" %% "akka-http"            % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-xml"        % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-stream"          % akkaVersion,
```

Comment:
That warning is expected, and it's not a problem as Akka 2.4.19 and 2.5.3 are binary compatible.